src/generics/generic-functions.md

```diff
@@ -19,7 +19,7 @@ fn pick<T>(n: i32, even: T, odd: T) -> T {
 
 fn main() {
     println!("picked a number: {:?}", pick(97, 222, 333));
-    println!("picked a tuple: {:?}", pick(28, ("dog", 1), ("cat", 2)));
+    println!("picked a tuple: {:?}", pick(28, "dog", "cat"));
```

Comment:
These aren't tuples anymore. I guess that would be `"picked a string"`. But the idea here was to show that T doesn't just have to be a named type like `String` or `i32`, but can be any type.

Reply:
I agree, I would add this as an additional example so the students see two "simple" types as well as a compound type.

Reply:
Hrm, I get what you're saying about demonstrating that generic types can also be compound types, though I don't think it's all that important to emphasize. I think students can generalize from "integers and strings" to "any type". But saying it explicitly to students might be good, so would it be good enough to add a speaker note about it?